packages/app/src/files-manager/components/files-manager/files-manager.component.tsx

```diff
@@ -63,6 +66,7 @@ const FilesManager: FunctionComponent<FilesManagerProps> = ({
     uploadingInfo: false,
     uploadingFailed: false,
   })
+  const dispatch = useDispatch()
```

Comment:
🟢 
We do not use the `useDispatch` hook in this project to no don't lose props control. 

However, it doesn't mean that we don't can start to use it. What do you think about discussing it at the front-end tech meeting?

Reply:
Sure we can talk about it, used it because it was the easiest way for me :) I'll take a while a think about it :)